docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[connecting-to-cloud]]
+=== Sending data to Elasticsearch Service
+
+Our hosted Elasticsearch Service is available on AWS, GCP, and Azure.
+{ess-trial}[You can try the Elasticsearch Service for free].
+
+Logstash comes with two settings that simplify sending data to
+https://cloud.elastic.co/[Elastic Cloud]: Cloud ID and Cloud Auth. 
```

Comment:
@jsvd double checking with you here, the cloud id/auth also works with ECE/ECK right?

Reply:
@kares Can you confirm?

Reply:
do not see why it wouldn't, I do not think there's anything special about cloud-id in ECE.
ECE docs already mentions setting up LS with cloud-id: https://www.elastic.co/guide/en/cloud-enterprise/current/ece-cloud-id.html#ece-cloud-id-logstash